kind/variant.lock

```diff
@@ -0,0 +1,4 @@
+dependencies:
+  kind:
+    version: 0.10.0
+    previousVersion: 0.10.0
```

Comment:
If the first time, will the previous version have the same value?

Reply:
Hmm. I'm not sure. What do you think @cw-sakamoto ?

Reply:
I always manually shift it by one, but I think the same is fine.